lib/neo4j/http/relationship_client.rb

```diff
@@ -25,10 +25,22 @@ def upsert_relationship(relationship:, from:, to:, create_nodes: false)
           CYPHER
         end
 
+        pk_attr_val = relationship.attributes["#{pk_attr}"] if pk_attr.present?
+        merge_relationship = ""
```

Comment:
this can probably be thrown into a helper method, but it does work this way!

Reply:
Or maybe we create a special `Neo4j::Http::RelationshipWithUuid` class that adds this? tbh I don't know why we wouldn't do this for all relationships because these should all be coming from join tables in 3md with a uuid column

Reply:
🤔 Idk if it makes sense for all relationships to have a `uuid`. Facility's `PARENT_OF` for example? I like `RelationshipWithUuid` 👍